Flick/Controllers/ListViewController.swift

```diff
@@ -202,7 +203,12 @@ extension ListViewController: UICollectionViewDelegateFlowLayout {
 extension ListViewController: MediaListHeaderDelegate, ModalDelegate {
 
     func addMedia() {
-        print("Add media")
+        // TODO: Remove show collaborator view later
+        addCollaboratorModalView = AddCollaboratorModalView()
+        addCollaboratorModalView.delegate = self
+        // TODO: Sends navigation bar to the back, but gets covered by the main view
+        navigationController?.navigationBar.layer.zPosition = -1
+        view.addSubview(addCollaboratorModalView)
```

Comment:
could we remove this for this PR? I'm working on adding media 😂 Will prob result in a merge conflict later